### PR TITLE
Rds civ pack

### PR DESCRIPTION
**Pull request by PSYKO-nz**
Ok, i think I got it this time.

if I haven't again sorry for wasting your time.

### CODE REVIEW

File: Missionframework/presets/civilians/RDSCiv.sqf [hunk 1]
```diff
@@ -0,0 +1,80 @@
+/*
+Needed Mods:
+- None
```
Comment (Wyqer) [inline Missionframework/presets/civilians/RDSCiv.sqf]: Please state the Mod, which is needed :)

**Comment by PSYKO-nz** [comment]
cool changes made, I copied the name of the steam workshop page so if someone searches that they will find it, does it also need the URL link?

**Comment by Wyqer** [comment]
The name is fine. Now it's all like it should be. Many thanks for the preset :)

**Comment by PSYKO-nz** [comment]
its the least I could do :) thank you for your work on this mission and your patience when dealing with people like me both here and on the forums, it been a really cool way to learn some new scripting and fill out the days, Hopefully, some others enjoy the preset, 

thanks, bro :)

**Pull request merged**